docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -116,11 +112,12 @@ __EOF__
 ----
 +
 <1> Set the replica count of the `manilaAPI` service to 3.
-<2> Set the replica count of the `manilaScheduler` service to 3.
-<3> Set the replica count of the `manilaShares` service to 1.
-<4> Ensure that the names of the back ends (`share_backend_name`) are the same as they were in {OpenStackShort} {rhos_prev_ver}.
-<5> Ensure that the appropriate storage management network is specified in the `networkAttachments` section. For example, the `manilaShares` instance with the CephFS back-end driver is connected to the `storage` network.
-<6> If you need to add extra files to any of the services, you can use `extraMounts`. For example, when using {Ceph}, you can add the {rhos_component_storage_file} Ceph user's keyring file as well as the `ceph.conf` configuration file.
+<2> If you use IPv6, change the load balancer IP to the load balancer IP in your environment, for example, `metallb.universe.tf/loadBalancerIPs: fd00:bbbb::80`.
+<3> Set the replica count of the `manilaScheduler` service to 3.
+<4> Set the replica count of the `manilaShares` service to 1.
+<5> Ensure that the names of the back ends (`share_backend_name`) are the same as they were in {OpenStackShort} {rhos_prev_ver}.
+<6> Ensure that the appropriate storage management network is specified in the `networkAttachments` section. For example, the `manilaShares` instance with the CephFS back-end driver is connected to the `storage` network.
```

Comment:
Maybe:
"Ensure that you specify the appropriate storage management network in the `networkAttachments` section."

just to make it less passive "is specified"